indico/modules/events/abstracts/templates/display/call_for_abstracts.html

```diff
@@ -0,0 +1,157 @@
+                        <time datetime="{{ dt.isoformat() }}">{{ dt | format_human_date }}</time>
+                    </div>
+                    <div>
+                        <div class="contrib-state small i-tag {{ mapping[abstract.public_state.name] }}">
+                            {{- abstract.state.title -}}
+                        </div>
+                        {% if abstract.accepted_track -%}
+                            <div class="contrib-track small">
+                                {{- abstract.accepted_track.title -}}
+                            </div>
+                        {%- endif %}
+                        {% if abstract.accepted_contrib_type -%}
+                            <div class="contrib-type small">
+                                {{- abstract.accepted_contrib_type.name -}}
+                            </div>
+                        {%- endif %}
+                    </div>
+                    {% if abstract.description -%}
+                        <a href="{{ url_for('abstracts.display_abstract', abstract) }}">
+                            <div class="description js-mathjax">
+                                {{- abstract.description|truncate(400) -}}
+                            </div>
+                        </a>
+                    {%- endif %}
+                </div>
+            {%- endfor %}
+        </div>
+    </section>
+{% endmacro %}
+
+{% block content %}
+    <div class="call-for-abstracts">
+        {% set cfa = event.cfa %}
+        {% set tz = event.display_tzinfo %}
+        {{ render_timeline_section(cfa) }}
+        {% if cfa.is_open and cfa.announcement %}
+            <section>
+                <div class="cfa-announcement">{{ cfa.announcement }}</div>
+            </section>
+        {% endif %}
+        <section class="cfa-infoline-section">
+            <div class="infoline announcement">
+                <div class="text">
+                    {%- if cfa.is_open %}
+                        <div class="label">{%- trans %}The call for abstracts is open{% endtrans -%}</div>
+                        <div>
+                            {%- trans %}Your submitted abstract will be subject to the reviewing process.{% endtrans -%}
+                        </div>
+                        <div class="toolbar right">
+                            <a class="i-button icon-plus highlight js-submit-abstract"
+                               data-href="{{ url_for('.submit', event) }}"
+                               data-method="GET"
+                               data-ajax>
+                                {%- trans %}Submit new abstract{% endtrans -%}
+                            </a>
+                        </div>
+                    {% else %}
+                        <div class="label">{% trans %}The call for abstracts is closed{% endtrans %}</div>
+                    {% endif -%}
+                </div>
+            </div>
+        </section>
+        {% if abstracts | length %}
```

Comment:
no need for `| length` (just like you usually don't use `len()` in python if you just want to know if it's not empty)